install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -3150,10 +3150,24 @@ authentication:
         enabled: true
         # -- SPIRE namespace to install into
         namespace: cilium-spire
+        # -- init container image of SPIRE agent and server
+        initImage:
+          override: ~
+          repository: "docker.io/library/busybox"
+          tag: "1.35.0"
+          digest: "sha256:223ae047b1065bd069aac01ae3ac8088b3ca4a527827e283b85112f29385fb1b"
+          useDigest: true
+          pullPolicy: "Always"
         # SPIRE agent configuration
         agent:
           # -- SPIRE agent image
-          image: ghcr.io/spiffe/spire-agent:1.6.3@sha256:8eef9857bf223181ecef10d9bbcd2f7838f3689e9bd2445bede35066a732e823
+          image:
+            override: ~
+            repository: "ghcr.io/spiffe/spire-agent"
+            tag: "1.6.3"
+            digest: "sha256:8eef9857bf223181ecef10d9bbcd2f7838f3689e9bd2445bede35066a732e823"
```

Comment:
Usually for these external dependencies, we would define them as exports in `install/kubernetes/Makefile.values` in order to simplify the updates, then in the `install/kubernetes/cilium/values.yaml.tmpl` file we have variables. You can scan up and look at examples around `HUBBLE_UI_BACKEND_REPO` (plus grep for this symbol in the codebase) to get a sense for how these are plumbed through.

I'd suggest using variable prefixes like the following and use the exact same pattern as other code in the template file uses:

- SPIRE_INIT_* for the busybox images,
- SPIRE_AGENT_* for the agent images,
- SPIRE_SERVER_* for the server images.

Reply:
sure, thanks